plugins/event_notification/providers/bpm/lib/api/filters/KalturaBusinessProcessServerFilter.php

```diff
@@ -5,11 +5,40 @@
  */
 class KalturaBusinessProcessServerFilter extends KalturaBusinessProcessServerBaseFilter
 {
+	/**
+	 * @var KalturaNullableBoolean
+	 */
+	public $currentDcOrNull;
+
+	/**
+	 * @var KalturaNullableBoolean
+	 */
+	public $currentDc;
+
 	/* (non-PHPdoc)
 	 * @see KalturaFilter::getCoreFilter()
 	 */
 	protected function getCoreFilter()
 	{
 		return new BusinessProcessServerFilter();
 	}
+
+	/* (non-PHPdoc)
+	 * @see KalturaFilter::toObject()
+	 */
+	public function toObject ( $object_to_fill = null, $props_to_skip = array() )
+	{
+		if(!$this->isNull('currentDc') && KalturaNullableBoolean::toBoolean($this->currentDc))
+			$this->dcEqual = kDataCenterMgr::getCurrentDcId();
+
+		elseif(!$this->isNull('currentDcOrNull') && KalturaNullableBoolean::toBoolean($this->currentDcOrNull))
+		{
+			if(kDataCenterMgr::getCurrentDcId())
```

Comment:
You can use more simple formula:
$this->dcGreaterThenOrNull = 1 - kDataCenterMgr::getCurrentDcId ()